.github/workflows/validate_esphome_beta.yml

```
@@ -11,6 +11,11 @@ on:  # yamllint disable-line rule:truthy
       - "*.c"
       - "*.cpp"
       - "*.py"
+    branches:
+      - main
+    paths-ignore:
```

actionlint (1.7.4)

both "paths" and "paths-ignore" filters cannot be used for the same event "push". note: use '!' to negate patterns

(events)